pyscript.core/types/stdlib/pyscript.d.ts

```diff
@@ -4,5 +4,9 @@ declare const _default: {
         "display.py": string;
     };
     "pyscript.py": string;
+    pyweb: {
```

Comment:
feel free to export `pyweb.pydom` and `pyweb.base.when` via `pyscript.py`  if you like, all you need is to define these exports in there:

```python
# ... whatever code in it
from pyweb import pydom
from pyweb.base import when
```

that allows `from pyscript import pydom, when` from users code.